addons/mail_bot/static/src/messaging/widget/notification_alert/notification_alert_tests.js

```diff
@@ -51,6 +51,7 @@ QUnit.test('notification_alert widget: display blocked notification alert', asyn
         },
     });
 
+    await nextAnimationFrame();
```

Comment:
This is going to be a long term mess I think.

I guess we should use `afterNextRender`, and create a version of it that resolves instead of rejecting when it is fine if nothing happens (for the 2 other tests).